### ISSUE TITLE
[WASM] Intermittent exception when changing layout

### ISSUE THREAD

**Issue by peternary** [unoplatform/uno#7008]
### Current behavior

In WASM, rearranging UI elements through c# will sometimes fail altogether.  The issue appears to be more frequent when the Uno application is under load or when the browser console is open, but neither of these conditions are necessary.

### Expected behavior

No exception should occur, as in UWP.

### How to reproduce it (as minimally and precisely as possible)

In the attached project, add a new tab and try to drag it to another position in the window.  This will often work, sometimes adding and rearranging more tabs will cause the issue to express, sometimes it is useful to reload the page and try again.  I can usually reproduce within 5 minutes.  When it fails, the tab and any control it was hovering over will disappear and exceptions will appear in the console.

Sample project: 
[TabAppearance.zip](https://github.com/unoplatform/uno/files/7137751/TabAppearance.zip)

Sample console output:
[localhost-1631200668526.log](https://github.com/unoplatform/uno/files/7137753/localhost-1631200668526.log)

### Workaround

n/a

### Works on UWP/WinUI

Yes

### Environment

Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia

### NuGet package version(s)

3.10.0-dev.685

### Affected platforms

WebAssembly

### IDE

Visual Studio 2019

**Comment by DierkDroth** [comment]
@jeromelaban have you been to reproduce the issue? Does it persist or should it be resolved?

**Comment by jeromelaban** [comment]
We have not been able to identify the cause yet. We'll let you know what we find.

**Comment by peternary** [comment]
I am still experiencing issues rearranging layout on latest Uno (version 4.0.0-dev.5452) with the sample project that had been provided.  Are any other package updates or configuration changes required?

**Comment by jeromelaban** [comment]
Which issue are you getting? Is it the same exception showing up the traces ?

**Comment by peternary** [comment]
It's failing to draw the new window placement.  I'm getting the following in the log:

>fail: Windows.UI.Input.GestureRecognizer[0]
      TabViewItem-124050 Inconsistent state, we already have a pending gesture for a pointer that is going down. Abort the previous gesture.

>VM344:1 fail: Windows.UI.Input.GestureRecognizer.Manipulation[0]
      Invalid manipulation state: We are receiving a down for the second time for the same pointer!This is however common when using iOS emulator with VNC where we might miss some pointer events due to focus being stole by debugger, in that case you can safely ignore this message.

**Comment by DierkDroth** [comment]
@jeromelaban would you have any more feedback on issue above?

**Comment by davidjohnoliver** [comment]
@peternary There's an issue with the TabView styling in the latest Fluent styles, I wonder if that could be what you're seeing? It will be fixed with [this PR](https://github.com/unoplatform/uno/pull/7456). You can work around it for now with:

```xml
<Application.Resources>
		<XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" ControlsResourcesVersion="Version1" />
	</Application.Resources>
```

Apart from that it seems to be working as expected on my end, so if you could provide some repro steps it would be appreciated.

**Comment by DierkDroth** [comment]
Thanks @davidjohnoliver. 

Unfortunately @peternary reports that the issue still is not resolved. He will get back with more details after he cleared his plate.

**Comment by DierkDroth** [comment]
@jeromelaban @davidjohnoliver
@peternary reports that the team still is held up by this intermittent issue on UNO 3.X. Does it make sense to upgrade to UNO 4.X and try again - however, this would require that [this issue](https://github.com/mono/SkiaSharp/pull/1873) is addressed - are do we have to invest more effort in order to fix UNO 3.X?

**Comment by jeromelaban** [comment]
@DierkDroth we're not able to reproduce the issue, would you have additional information that you can provide with the current sample? Uno 4.0.x won't significantly change the behavior of this issue at this time.

We're following up with the team maintaining SkiaSharp to get updates.

**Comment by jeromelaban** [comment]
I'm going to open another issue related to the new exception you've provided above, as it's not related to the original issue.
